src/ace/managers/advancedsprite.c

```diff
@@ -0,0 +1,268 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ #include <ace/managers/advancedsprite.h>
+ #include <ace/utils/bitmap.h>
+ #include <ace/managers/blit.h>
+ #include <ace/utils/sprite.h>
+ #include <ace/managers/system.h>
+ 
+ 
+ #define SPRITE_WIDTH 16
+ #define TWOBPP_BYTEWIDTH 2
+ 
+ void advancedSpriteManagerCreate(const tView *pView, UWORD uwRawCopPos) {
```

Comment:
Perhaps we don't need `advancedSpriteManagerCreate()` and `advancedSpriteManagerDestroy()` and just instruct the user to create the regular sprite manager first? This would also make using tSprite and tAdvancedSprite at the same time more intuitive, since not every sprite might need this functionality and you could squeeze out more objects this way.

Perhaps we could even move the tAdvancedSprite and its functions to managers/sprite.c and make a natural part of it?

Reply:
Ok for me.

Just pushed the modification.